.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+### Checklist
+
+* [ ] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
+* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
+* [ ] At least one test case is included for this feature or bug fix
```

Comment:
I don't think this check list for test case is necessary.

Reply:
fixed it.